in-relentless-pursuit-of-rest.md

```diff
@@ -0,0 +1,31 @@
+# In Relentless Pursuit of REST
+
+## Abstract
+
+"That's not very RESTful." As a Rails developer you've probably heard or even
+spoken that proclamation before, but what does it really mean? What's so great
+about being RESTful anyway?
+
+In this talk, you'll learn to refactor existing code to follow RESTful
+principles. Together we'll discuss the changes and the clarity of purpose they
+bring and pursue follow-on refactorings that resourceful thinking make more
+comfortable.
+
+## Details
+
+## Pitch
+
+Each year I familiarize myself with many Rails applications and the teams that
+produce them. I continue to see controllers with 10 or more actions, controllers
+that have a 1:1 coupling with ActiveRecord models, and routes files cluttered
+with custom `member` and `collection` routes.
+
+Rails *can be* RESTful, but we've done a poor job communicating how to pursue
+this. We added `resources` to the router, had a brief chat about it, and moved
+on as if that battle were won and need not be revisited.
+
+I think there are a good number of developers who hear, "That's not very
```

Comment:
What do you think about having the third paragraph first, then the second, and cut the first?